packages/block-library/src/navigation-submenu/edit.js

```diff
@@ -0,0 +1,637 @@
+						) }
+					/>
+					<TextControl
+						value={ title || '' }
+						onChange={ ( titleValue ) => {
+							setAttributes( { title: titleValue } );
+						} }
+						label={ __( 'Link title' ) }
+						autoComplete="off"
+					/>
+					<TextControl
+						value={ rel || '' }
+						onChange={ ( relValue ) => {
+							setAttributes( { rel: relValue } );
+						} }
+						label={ __( 'Link rel' ) }
+						autoComplete="off"
+					/>
+				</PanelBody>
+			</InspectorControls>
+			<div { ...blockProps }>
+				{ /* eslint-disable jsx-a11y/anchor-is-valid */ }
+				<ParentElement className="wp-block-navigation-item__content">
+					{ /* eslint-enable */ }
+					{
+						<RichText
+							ref={ ref }
+							identifier="label"
+							className="wp-block-navigation-link__label"
+							value={ label }
+							onChange={ ( labelValue ) =>
+								setAttributes( { label: labelValue } )
+							}
+							onMerge={ mergeBlocks }
+							onReplace={ onReplace }
+							aria-label={ __( 'Navigation link text' ) }
+							placeholder={ itemLabelPlaceholder }
+							withoutInteractiveFormatting
+							allowedFormats={ [
+								'core/bold',
+								'core/italic',
+								'core/image',
+								'core/strikethrough',
+							] }
+						/>
+					}
+					{ isLinkOpen && (
+						<Popover
+							position="bottom center"
+							onClose={ () => setIsLinkOpen( false ) }
+							anchorRef={ listItemRef.current }
+						>
+							<KeyboardShortcuts
+								bindGlobal
+								shortcuts={ {
+									escape: () => setIsLinkOpen( false ),
+								} }
+							/>
+							<LinkControl
+								className="wp-block-navigation-link__inline-link-input"
+								value={ link }
+								showInitialSuggestions={ true }
+								withCreateSuggestion={ userCanCreate }
+								createSuggestion={ handleCreate }
+								createSuggestionButtonText={ ( searchTerm ) => {
+									let format;
+									if ( type === 'post' ) {
+										/* translators: %s: search term. */
+										format = __(
+											'Create draft post: <mark>%s</mark>'
+										);
+									} else {
+										/* translators: %s: search term. */
+										format = __(
+											'Create draft page: <mark>%s</mark>'
+										);
+									}
+									return createInterpolateElement(
+										sprintf( format, searchTerm ),
+										{ mark: <mark /> }
+									);
+								} }
+								noDirectEntry={ !! type }
+								noURLSuggestion={ !! type }
+								suggestionsQuery={ getSuggestionsQuery(
+									type,
+									kind
+								) }
+								onChange={ ( updatedValue ) =>
+									updateNavigationLinkBlockAttributes(
+										updatedValue,
+										setAttributes,
+										attributes
+									)
+								}
+								onRemove={ () => {
+									setAttributes( { url: '' } );
+									speak( __( 'Link removed.' ), 'assertive' );
+								} }
+							/>
+						</Popover>
+					) }
+					{ showSubmenuIcon && (
+						<span className="wp-block-navigation-link__submenu-icon">
```

Comment:
It would be good to avoid using `link` in the classname. Is it possible to have a more normalized classname for this element? I notice `item` is being used for elements that are in the link and submenu, so maybe that.

Reply:
Oh, missed this one! It should have been changed to `wp-block-navigation__submenu-icon`, I'll do that now.

Reply:
Fixed!